proto/prehog/v1alpha/teleport.proto

```diff
@@ -381,15 +381,22 @@ message SubmitEventRequest {
     SFTPEvent sftp = 37;
     AgentMetadataEvent agent_metadata_event = 38;
   }
+
   reserved 8; // UIOnboardGetStartedClickEvent
   reserved "ui_onboard_get_started_click";
 }
 message SubmitEventResponse {}
 
+message SubmitEventsRequest {
+  repeated SubmitEventRequest events = 1;
+}
+message SubmitEventsResponse {}
+
 message HelloTeleportRequest {}
 message HelloTeleportResponse {}
 
 service TeleportReportingService {
   rpc SubmitEvent(SubmitEventRequest) returns (SubmitEventResponse) {}
```

Comment:
Are we still using SubmitEvent, or should we deprecate it?

Reply:
Good point, we're not using `SubmitEvent` anymore (and if we needed a single event submit we could just send a single event through `SubmitEvents` anyway).

I added a deprecation notice in gravitational/cloud#3767, it'll get in here the next time we sync the proto.